repo/mk/js.test.cypress.mk

```diff
@@ -0,0 +1,76 @@
+# Adds 'cypress/integration/**/*.js' targets to run jest on a specific cypress/integration/**/*.js file.
+#
+# ------------------------------------------------------------------------------
+#
+# Adds a 'test-cypress' internal target to run all SF_CYPRESS_TEST_FILES (defaults to cypress/integration/**/*.js).
+# The 'test-cypress' target is automatically included in the 'test' target via SF_TEST_TARGETS.
+#
+# The arguments to the cypress executable can be changed via CYPRESS_ARGS.
+# Cypress will by default run on the url defined in its cypress.json file.
+# To run it on another url define WEB_BASE_URL before running the target.
+#
+# ------------------------------------------------------------------------------
+
+WAIT_ON = $(call npm-which,WAIT_ON,wait-on)
+
+KILL_PORT = $(call npm-which,KILL_PORT,kill-port)
+
+CYPRESS = $(call npm-which,CYPRESS,cypress)
+
+CYPRESS_BROWSER_TARGET ?= chrome
+
+CYPRESS_ARGS += \
+	--browser $(CYPRESS_BROWSER_TARGET) \
+
+CYPRESS_PORT ?= 4200
+
+CYPRESS_WEB_URL ?= http://localhost
+
+
+SF_CYPRESS_TEST_FILES += \
+	$(shell $(FIND_Q_NOSYM) cypress/integration -type f -name "*.js" -print) \
+
+SF_VENDOR_FILES_IGNORE += \
```

Comment:
Do we still need this variable?

Reply:
🤦‍♂ 😂 "Andrei this, Andre that"

line 33 doesn't have a purpose, that's why it could/should be removed 😉

Reply:
🤦‍♂ 😂 "Andrei this, Andre that"

line 33 doesn't have a purpose, that's why it could/should be removed 😉